scripts/build/macos-standalone.sh

```diff
@@ -0,0 +1,122 @@
+project_file=swift/apple/Firezone.xcodeproj
+codesign_identity="Developer ID Application: Firezone, Inc. (47R2M6779T)"
+
+if [ "${CI:-}" = "true" ]; then
+    # Configure the environment for building, signing, and packaging in CI
+    setup_runner \
+        "$STANDALONE_MACOS_APP_PROVISIONING_PROFILE" \
+        "$app_profile_id.provisionprofile" \
+        "$STANDALONE_MACOS_NE_PROVISIONING_PROFILE" \
+        "$ne_profile_id.provisionprofile"
+fi
+
+# Build and sign
+insert_build_timestamp "$project_file/project.pbxproj"
+
+echo "Building and signing app..."
+xcodebuild build \
+    GIT_SHA="$git_sha" \
+    CODE_SIGN_STYLE=Manual \
+    CODE_SIGN_IDENTITY="$codesign_identity" \
+    PACKET_TUNNEL_PROVIDER_SUFFIX=-systemextension \
+    CONFIGURATION_BUILD_DIR="$temp_dir" \
+    APP_PROFILE_ID="$app_profile_id" \
+    NE_PROFILE_ID="$ne_profile_id" \
+    ONLY_ACTIVE_ARCH=NO \
+    -project "$project_file" \
+    -skipMacroValidation \
+    -configuration Release \
+    -scheme Firezone \
+    -sdk macosx \
+    -destination 'platform=macOS'
+
+# Create disk image
+mkdir -p "$dmg_dir/.background"
+mv "$temp_dir/Firezone.app" "$dmg_dir/Firezone.app"
+cp "scripts/build/dmg_background.png" "$dmg_dir/.background/background.png"
+ln -s /Applications "$dmg_dir/Applications"
+hdiutil create \
+    -volname "Firezone Installer" \
+    -srcfolder "$dmg_dir" \
+    -ov \
+    -format UDRW \
+    "$package_path"
+
+# Mount disk image for customization
+mount_dir=$(hdiutil attach "$package_path" -readwrite -noverify -noautoopen | grep -o "/Volumes/.*")
+
+# Embed background image to instruct user to drag app to /Applications
+osascript <<EOF
+tell application "Finder"
+    tell disk "Firezone Installer"
+        open
+        set current view of container window to icon view
+        set toolbar visible of container window to false
+        set statusbar visible of container window to false
+        set bounds of container window to {100, 100, 800, 400}
+        set viewOptions to the icon view options of container window
+        set arrangement of viewOptions to not arranged
+        set icon size of viewOptions to 128
+        set background picture of viewOptions to file ".background:background.png"
+        set position of item "Firezone.app" of container window to {200, 128}
+        set position of item "Applications" of container window to {500, 128}
+        close
+        open
+        update without registering applications
+        delay 2
+    end tell
+end tell
+EOF
+
+# Unmount disk image
+hdiutil detach "$mount_dir"
+
+# Convert to read-only
+hdiutil convert "$package_path" -format UDZO -o "$dmg_path"
+
+# Sign disk image
+codesign --force --sign "$codesign_identity" "$dmg_path"
+
+echo "Disk image created at $dmg_path"
+
+# Notarize disk image and embedded app
+if [ "$notarize" = "true" ]; then
+    private_key_path="$temp_dir/firezone-api-key.p8"
+    echo -n "$API_KEY" | base64 --decode -o "$private_key_path"
```

Comment:
This could reuse the other function in `lib` if you could just pass a single output path.